Dockerfile

```diff
@@ -26,12 +29,11 @@ RUN useradd -d /home/seleuser -m seleuser
 RUN mkdir -p /home/seleuser/chrome
 RUN chown -R seleuser /home/seleuser
 RUN chgrp -R seleuser /home/seleuser
-# fix https://code.google.com/p/chromium/issues/detail?id=318548
-RUN mkdir -p /usr/share/desktop-directories
+
 ADD ./scripts/ /home/root/scripts
-RUN npm install -g \
-  selenium-standalone@3.0.2 \
```

Comment:
You're right, the dependencies were renamed but it's way more stable to use specific versions, I'll add those back.